### ISSUE TITLE
so sorry but another problem when i start fortnite

### ISSUE THREAD

**Issue by whoiscamm** [RareDevs/Rare#319]
![another problem](https://github.com/RareDevs/Rare/assets/152131406/a5aceb4d-bfe3-47c2-9c07-6b02df0b48c2)

**Comment by whoiscamm** [comment]
damn well ig ill test my luck

**Comment by whoiscamm** [comment]
hey quick question when i download sum wht does it mean if it pauses by it self and just stops

**Comment by loathingKernel** [comment]
I am not sure I understand the question. I guess it means that it finished, but you have to better describe the behaviour.

**Comment by whoiscamm** [comment]
well say im downloading and it stops at like 35 and js doesn't move for the rest of the time and never finishes

**Comment by loathingKernel** [comment]
The progress of extracting the downloaded data is not visible. You have to just wait for it. In any case run Rare with the `--debug` argument and try to reproduce it. If it happens again upload the log file here. You can find logs in the following location `C:\Users\<username>\AppData\Local\Rare\Rare\cache/logs\`

**Comment by whoiscamm** [comment]
u want me to upload it here so u can see? and sorry but i don't think i know how to run rare with the debug argument

**Comment by whoiscamm** [comment]
so do i right click on rare create shorcut and add debug at the end

**Comment by whoiscamm** [comment]
[Rare_23-12-02--16-12.log](https://github.com/RareDevs/Rare/files/13540065/Rare_23-12-02--16-12.log)
[Rare_23-12-02--16-32.log](https://github.com/RareDevs/Rare/files/13540066/Rare_23-12-02--16-32.log)
[Rare_23-12-02--16-39.log](https://github.com/RareDevs/Rare/files/13540067/Rare_23-12-02--16-39.log)
[Rare_23-12-02--17-31.log](https://github.com/RareDevs/Rare/files/13540068/Rare_23-12-02--17-31.log)
[Rare_23-12-03--01-01.log](https://github.com/RareDevs/Rare/files/13540069/Rare_23-12-03--01-01.log)
[Rare_23-12-03--14-23.log](https://github.com/RareDevs/Rare/files/13540070/Rare_23-12-03--14-23.log)
[Rare_23-12-03--14-32.log](https://github.com/RareDevs/Rare/files/13540071/Rare_23-12-03--14-32.log)
[Uploading Rare_23-12-03--14-34.log…]()

**Comment by loathingKernel** [comment]
No, you don't. Also, these questions might be better off in the Discussions section, not the Issues section of GitHub. If the issue with fortnite is resolved, I would like to close this issue.
